src/style.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: Avenir;
+  font-style: book;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
```

Comment:
Consider using a more descriptive font-style value like 'bold' instead of 'heavy'.